hugo_site/content/live.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Schedule
+date: 2018-12-16T18:48:39+01:00
+draft: false
+type: "single"
+description: "DjangoCon Europe 2019 LIVE - stream and ask questions"
+menu:
+  main:
+    parent: "schedule"
+    name: "Live stream and questions"
+---
+
+# Live stream
+
+Tune in here: https://www.youtube.com/user/djangoconeurope
+
+# Questions
+
+During talks, a host will select questions from the internet crowd. You can tag your question
+Twitter with `djangoconQA` or send a question on the Freenode channel `#djangocon`.
+
+Questions asked are subject to our [Code of Conduct](/conduct/) and any
+perceived violations will be filtered out.
+
+## Asking good questions
+
+If a speaker takes questions after a talk, we encourage you to come to the mic 🎤
+placed in the room. We will also take questions from internet audiences if time
+permits. Please ask questions that are constructive and supportive:
+
+* **A good question**: Is one that seeks to explore the topic of the talk;
```

Comment:
```suggestion
* **A good question**: is not a comment, and ends in question mark; Is one that seeks to explore the topic of the talk;
```

Reply:
Haha, oh YES; that one! It should seem obvious, but I've seen it necessary to repeat sooo many times :) THANKS!